README.md

```diff
@@ -1,31 +1,60 @@
+The OpenAPI Specification defines a standard, language-agnostic interface description for REST APIs which allows both humans and computers to discover and understand the capabilities of the service without access to source code, additional documentation, or through network traffic inspection.  When properly defined via OpenAPI, a consumer can understand and interact with the remote service with a minimal amount of implementation logic.  Similar to what interface descriptions have done for lower-level programming, the OpenAPI Specification removes the guesswork in calling the service.
 
-Development of the next version of the OpenAPI Specification is guided by the [OAI Technical Contributors Board](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/CONTRIBUTORS.md).  This group of committers bring their API expertise, incorporate feedback from the community, and expand the group of committers as appropriate.  All development activity on the future specification will be performed as features and merged into this branch.  Upon release of the OpenAPI Specification, this branch will be merged to master.
+Use cases for machine-readable API interfaces include interactive documentation; code generation for documentation, client, and server; and automated test cases. OpenAPI descriptions describe APIs via YAML or JSON documents that adhere to the OpenAPI Specification.  These documents can either be produced and served statically, or be generated dynamically from your application.
 
-The current process for development of the OpenAPI Specification is described in [Development Guidelines](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/DEVELOPMENT.md).
+The OpenAPI Specification does not require you to rewrite your existing API.  It does not require binding any software to a service--the service being described may not even be yours.  It does, however, require the capabilities of the service be described in the structure of the OpenAPI Specification.  Not all services can be described by OpenAPI--this specification is not intended to cover every possible use-case of a REST API. The OpenAPI Specification does not define a specific development process such as design-first or code-first.  It does facilitate either technique by establishing clear interactions with a REST API.
+
+This GitHub project is the starting point for OpenAPI.
+Here you will find the information you need about the OpenAPI Specification, simple examples of what it looks like,
+and some general information regarding the project.
+
+## Current Version - 3.0
+
+The current version of the OpenAPI specification is [OpenAPI Specification 3.0](versions/3.0.md).
+
+### Previous Versions
+
+This repository also contains the [OpenAPI Specification 2.0](versions/2.0), which is identical to the Swagger 2.0 specification, 
+as well as the Swagger 1.2 and Swagger 2.0 specifications.
```

Comment:
Technically it only currently contains one copy of the OpenAPI 2.0 specification, not the Swagger 2.0 specification as well.